packages/tools/pie-icons-webc/README.md

```diff
@@ -27,11 +27,17 @@ yarn add @justeattakeaway/pie-icons-webc
 
 ```js
 // Only import what you need!
-import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant';
+import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant'; // Import the typescript version of the icon
+// OR
+import '@justeattakeaway/pie-icons-webc/dist/icons/IconAppRestaurant.js'; // Import the ESM version of the icon
```

Comment:
I'm guessing you wouldn't need to define the `.js` here unless writing TS in your app? In which case, you'd likely just use the TS import above.

If so, I'd probably remove the `.js` here.

Also, if we were to compile the JS files to the icons directory (instead of the `dist/icons` directory), wouldn't that make the import path simpler (as it would be the same potentially I'd think and get handled by the bundler?)

Reply:
I was actually leaning the other way and considered removing the TS instructions!
I tried to import the TS versions in pie-modal but that was breaking on CI, saying it [couldn't resolve the import](https://github.com/justeattakeaway/pie/actions/runs/5712859793/job/15477451479#step:5:135). The JS works just fine.

Reply:
I was mainly just wondering if we outputted everything to the `icons` folder, if we could just get rid of the `dist` folder entirely. It would make the import statements a bit cleaner then (if that works as expected).

What's the use-case for importing the TS icons out of interest?